File: Go/models/group.go
package models

import (
	"go.mongodb.org/mongo-driver/bson/primitive"
)

type Group struct {
	Id     primitive.ObjectID `json:"id,omitempty" bson:"id"`
	Name   string             `json:"name,omitempty" bson:"name"`
	UserId primitive.ObjectID `json:"userid,omitempty" bson:"userid"`
}

File: vue-client/src/classes/group.ts
import {Entity} from "@/interfaces/entity";

export class Group implements Entity {
    id: number;
    name: string;
    userId: string;
   

    constructor(id = 0, name = '', userId = '') {
        this.id = id;
        this.name = name;
        this.userId = userId;

    }
}